### ISSUE TITLE
move contribute page from github wiki to website?

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit-project.github.io#90]
It's jarring to have top-level navigation switch sites.

We should probably move the contribute page to the website.

It's not updated frequently (last update was in 2015), and we could basically have a simple script that fetches the raw file and appends a Frontmatter header, if we want it automated (or simply run it once in a while, when the file changes).

```
---
title: Contributing
base: https://github.com/cockpit-project/cockpit/wiki/
---
```

(I could easily extend the layout to support a base URL, so all relative links would point to the right area.)

Raw file is available @ https://raw.github.com/wiki/cockpit-project/cockpit/Contributing.md

---

Or, we could just copy the file as-is and make any necessary tweaks and not overengineer it. (;

**Comment by garrett** [comment]
Alternatively, we could do something like a `git subtree` and include all of the GitHub wiki on the website and link to the local version.

**Comment by garrett** [comment]
Fixed in #103
